.rubocop.yml

```diff
@@ -1,6 +1,15 @@
 inherit_from:
   - https://raw.githubusercontent.com/DFE-Digital/apply-for-teacher-training/master/.rubocop.yml
 
+# Disabled because this project doesn’t use GOV.UK Design System Form Builder
+Govuk/GovukSubmit:
+  Enabled: false
+
+Govuk/GovukLinkTo:
+  Exclude:
+    # link_to in footer
+    - 'app/views/layouts/application.html.erb'
```

Comment:
Is this cop not inherited from Apply (line 2)?

Reply:
Do you mean is `'app/views/layouts/application.html.erb'` already excluded in the Apply config? In which case no, because Apply has separate partials for the footer.

Reply:
👌